readme.md

```diff
@@ -94,15 +106,13 @@ WordPress has a [whitelist of file extensions that it allows in uploads](https:/
 
 ## Authentication and authorization 
 
-Supports the full OAuth2/IndieAuth authentication and authorization flow. Defaults to IndieAuth. Custom auth and token endpoints can be used by overriding the `MICROPUB_AUTHENTICATION_ENDPOINT` and `MICROPUB_TOKEN_ENDPOINT` endpoints or by setting the options `indieauth_authorization_endpoint` and `indieauth_token_endpoint`.
+For reasons of security it is recommended that you only use this plugin on sites that implement HTTPS.
 
-If the token's `me` value matches a WordPress user's or author post URL, that user will be used. If there is only one site author that will be matched otherwise.
+Supports the full OAuth2/IndieAuth authentication and authorization flow. Defaults to IndieAuth.com. Custom auth and token endpoints can be used by overriding the `MICROPUB_AUTHENTICATION_ENDPOINT` and `MICROPUB_TOKEN_ENDPOINT` endpoints or by setting the options `indieauth_authorization_endpoint` and `indieauth_token_endpoint`.
 
-Alternatively, you can set `MICROPUB_LOCAL_AUTH` to 1 to disable the plugin's authorization function, for example if you want authorization to be done by WordPress or another plugin. It will also 
-be disabled if the IndieAuth plugin is installed.
-
-Finally, for ease of development, if the WordPress site is running on `localhost`, it logs a warning if the access token is missing or invalid and still allows the request.
```

Comment:
does this PR remove this feature? or was it removed before? it's useful, and i think harmless; i'd like to keep it.

Reply:
It replaces using it with a filter, which means more in the plugin territory than user territory

Reply:
...but also removes the existing built in behavior, and requires users to write PHP code (ie add a filter handler) to get it back. at minimum we'd need to document that...but i'd actually just like to keep the existing behavior, if that's ok.

mind adding it back in a separate PR? (since we'd probably only merge this one if it fixes @miklb's problem, which seems uncertain.)

feel free to grab me on IRC to talk more easily!